projects/ngneat/transloco/src/lib/transloco.service.ts

```diff
@@ -555,10 +553,23 @@ export class TranslocoService implements OnDestroy {
     this.failedLangs.clear();
   }
 
-  private handleFailure(lang: string, mergedOptions) {
+  private handleFailure(lang: string, loadOptions: LoadOptions) {
+    // When starting to load a first choice language,
```

Comment:
`// When starting to load a first choice language, initialize the failed counter and resolve the fallback langs`

Reply:
@shaharkazaz Fixed. Thanks!